mobile/android/device-android-debug-bridge.ts

```diff
@@ -6,7 +6,7 @@ interface IComposeCommandResult {
 }
 
 export class DeviceAndroidDebugBridge extends AndroidDebugBridge implements Mobile.IDeviceAndroidDebugBridge {
-	constructor(private identifier: string,
+	constructor(public identifier: string,
```

Comment:
I believe you do not need this change as the only place where you are using the property outside of the current class is on a place, where you already have the identifier.